src/control-plane/backend/lambda/api/router/reporting.ts

```diff
@@ -56,6 +56,12 @@ reporting_project.post(
     return reportingServ.warmup(req, res, next);
   });
 
+reporting_project.post(
```

Comment:
@tyyzqmf should we add this route to middleware to grant the permission for the role analyst?

Reply:
The rule already includes this route: `routerRoles.set('ALL /api/metadata/*', [IUserRole.ADMIN, IUserRole.ANALYST])`